ts/src/base/Exchange.ts

```diff
@@ -7010,6 +7010,39 @@ export default class Exchange {
         return [ request, params ];
     }
 
+    handleSinceUntilWithDiapason (request: any, params: any, sinceKey: string, untilKey: string, since: Int, maxDaysBetween: number) {
```

Comment:
this name is not intuitive, also this method maybe be hard to use if we have extra logic besides the distance between since and logic.

What if instead we add a method that calculates the since/until given the until/since + distance.

Like `sumDaysToTimestamp` or something that you could use by doign

```
const sinceWithDistance = this.sumDaysToTimestamp(ts, -7);
const daysFromNow = this.sumDaysToTImestamp(ts, 7)
```

Reply:
I think i'll wait for other exchange implementations too, when I see this pattern might be re-occuring, then i'll think moving to base and making it uniform.
at this moment, i'll placed back in bitget.ts locally and i think it should now be ok, wdyt?